src/Microsoft.SqlTools.ServiceLayer/EditData/EditColumnMetadata.cs

```diff
@@ -36,6 +36,22 @@ public EditColumnMetadata()
         /// </summary>
         public string EscapedName { get; set; }
 
+        /// <summary>
+        /// The column's expression for select statement
+        /// </summary>
+        public string ExpressionForSelectStatement
+        {
+            get
+            {
+                return IsHierarchyId ? string.Format("{0}.ToString() {0}", EscapedName) : EscapedName;
```

Comment:
Using string interpolation here would be nice
I suspect there will be other column types that will use similar logic (Geometry type perhaps?). If so, let's double check that this pattern is scalable

Reply:
I know this is implicit, but it's really nice to have the explicit `AS` added between the two